app/models/patient.rb

```diff
@@ -78,6 +78,10 @@ def latest_phone_number
     phone_numbers.last&.number
   end
 
+  def latest_mobile_number
+    phone_numbers.where(phone_type: 'mobile').last&.number
```

Comment:
```suggestion
    phone_numbers.where(phone_type: 'mobile').order(device_created_at: :desc).first
```

Reply:
Maybe also consider making the `where` a scope on PhoneNumber itself.

Reply:
@govindkrjoshi There's a default scope on PhoneNumber which orders asc. The query effectively looks like: 
```ruby
phone_numbers.unscope(:order).mobile.order(device_created_at: :desc).first&.number
```

Should I try and get rid of the default scope?